variables.tf

```diff
@@ -5,6 +5,9 @@ variable "data_ingest_cidr_block" {}
 variable "peering_cidr_block" {}
 variable "az" {}
 variable "data_landing_bucket" {}
+variable "s3_bucket_name" {}
+variable "s3_bucket_acl" {}
```

Comment:
won't this always be private? does it need to be a var?

Reply:
it may not necessary be set to private: https://docs.aws.amazon.com/AmazonS3/latest/dev/acl-overview.html#canned-acl

Reply:
so we can remove the param entirely?

Reply:
actually for this bucket it is not required. removing it.